tests/Nelmio/Alice/Fixtures/Parser/Methods/PhpTest.php

```diff
@@ -110,4 +110,31 @@ public function testIncludeFiles()
         ];
         $this->assertEquals($expectedData, $data);
     }
+
+    public function testIncludeFilesOrder()
+    {
+        $data = $this->parser->parse(__DIR__.'/../../../support/fixtures/include_order.php');
+
+        $expectedData = [
+            'Foo' =>
+                [
+                    'foo' =>
+                        [
+                            'id' => 1,
+                            'text' => '<word()>',
+                        ],
+                ],
+            'Bar' =>
+                [
+                    'bar' =>
+                        [
+                            'id' => 1,
+                            'text' => '<@foo->text>'
+                        ],
+                ],
+        ];
+
+        $this->assertEquals($expectedData, $data);
+        $this->assertTrue($expectedData === $data);
```

Comment:
>it will not check if the order is the same.

But `[1,2] == [2,1]` will return `false`, regardless of if you are using `==` or `===`. That's why I'm a bit lost with your explanation.